src/samples/article-history/v1/one-version-with-preprint.json

```diff
@@ -0,0 +1,27 @@
+{
+    "received": "2016-01-02",
+    "accepted": "2016-02-28",
+    "versions": [
+        {
+            "status": "vor",
+            "id": "14107",
+            "version": 1,
+            "type": "research-article",
+            "doi": "10.7554/eLife.14107",
+            "authorLine": "Yongjian Huang et al.",
+            "title": "Molecular basis for multimerization in the activation of the epidermal growth factor",
+            "stage": "published",
+            "published": "2016-03-28T00:00:00Z",
+            "versionDate": "2016-03-28T00:00:00Z",
+            "statusDate": "2016-03-28T00:00:00Z",
+            "volume": 5,
+            "elocationId": "e14107"
+        },
+        {
+            "type": "preprint",
+            "description": "This manuscript was published as a preprint at bioRxiv.",
+            "uri": "https://www.biorxiv.org/content/10.1101/2019.08.22.6666666v1",
```

Comment:
note this is a version-specific URL, and bioRxiv preprints can have multiple versions too (which this API is probably not interested in capturing)

Reply:
it's a fair point and something @FAtherden-eLife might have thought about already. As far as the spec and elife-tools are concerned we're just using whatever we pull out of the XML.

Reply:
> (which this API is probably not interested in capturing)

Yes, we're not interested in capturing the different versions.

> note this is a version-specific URL

We may use the DOI uri instead (i.e. `https://doi.org/10.1101/2019.08.22`) which will resolve to the latest version. Since it'll still be a resolvable uri, presumably this won't cause any problems here?

Reply:
nah, no problems. I'll update the example so it's less confusing.